rules/TypeDeclaration/Rector/Class_/ReturnTypeFromStrictTernaryRector.php

```diff
@@ -68,119 +73,52 @@ public function getValue($number): int
      */
     public function getNodeTypes(): array
     {
-        return [Class_::class];
+        return [ClassMethod::class, Function_::class, Closure::class];
     }
 
     /**
-     * @param Class_ $node
+     * @param ClassMethod|Function_|Closure $node
      */
     public function refactorWithScope(Node $node, Scope $scope): ?Node
     {
-        $hasChanged = false;
-
-        foreach ($node->getMethods() as $classMethod) {
-            if ($classMethod->returnType instanceof Node) {
-                continue;
-            }
-
-            $onlyStmt = $classMethod->stmts[0] ?? null;
```

Comment:
before this PR this rector only supported call-methods with a single line.
I wided the scope to call-likes with a single return